### ISSUE TITLE
USER ISSUE: 

### ISSUE THREAD

**Issue by InzeNL** [StrangeLoopGames/EcoIssues#5859]
**Version:** 0.7.2.0 beta

**Steps to Reproduce:**
1. Smack a hammer at a campfire

**Expected behavior:**
No crash

**Actual behavior:**
The following error message popped up:
```
Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Object reference not set to an instance of an object.


Stack:
   at Eco.Gameplay.Items.Inventory.MoveAsManyItemsAsPossible(Inventory destination, Func`2 sourceStackPredicate, User user)
   at Eco.Gameplay.Components.StorageComponent.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Networking.NetPort.Update()
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>
```

**Comment by InzeNL** [comment]
Clicking it with left mousebutton will crash it (even without a hammer)

**Comment by InzeNL** [comment]
![image](https://user-images.githubusercontent.com/2505862/37246922-a8801880-24b2-11e8-811d-f5bdaa717415.png)

Note the campfire in the center

**Comment by InzeNL** [comment]
Probable cause: I was holding Hewn logs, by left clicking I would be taking the logs from the campfire.
After removing the Hewn logs from my inventory no crash occured

**Comment by MarjieAVolk** [comment]
This is fixed in 0.7.2.3
